mungegithub/mungers/size.go

```diff
@@ -275,18 +266,7 @@ func calculateSize(adds, dels int) string {
 	return sizeXXL
 }
 
-func (s *SizeMunger) isStaleIssueComment(obj *github.MungeObject, comment *githubapi.IssueComment) bool {
-	if !obj.IsRobot(comment.User) {
-		return false
-	}
-	stale := sizeRE.MatchString(*comment.Body)
-	if stale {
-		glog.V(6).Infof("Found stale SizeMunger comment")
-	}
-	return stale
-}
-
 // StaleIssueComments returns a slice of stale issue comments.
 func (s *SizeMunger) StaleIssueComments(obj *github.MungeObject, comments []*githubapi.IssueComment) []*githubapi.IssueComment {
```

Comment:
This function can be removed completely and `RegisterStaleIssueComments(s)` can be removed from the `init` function.